### DISCUSSION TITLE
Enable SSL, documentation please

### DISCUSSION THREAD

**Discussion by sunkararp** [vesoft-inc/nebula#4303] [Q&A]
@wey-gu 

How to enable SSL; please provide proper documentation

This is what was provided in the official doc [link](https://docs.nebula-graph.io/3.1.0/7.data-security/4.ssl/). please provide some examples that make sense.

**Comment by xiajingchun** [comment]
Could you be more specific about what kind of examples you're looking for? 
If you meant how to get the self-signed certs files, you might need to check out openssl or keytool 4 java, etc.
According to the official doc you've already found, there're 3 different kinds of policies that you'll need to decide which one to take.

**Reply by sunkararp** [reply]
> Could you be more specific about what kind of examples you're looking for? If you meant how to get the self-signed certs files, you might need to check out openssl or keytool 4 java, etc. According to the official doc you've already found, there're 3 different kinds of policies that you'll need to decide which one to take.

@xiajingchun thanks for the reply.

I need to more details on various SSL configurations..such as cert subject name?

cert_path & key_apth: What is the recommended location? Can we use K8S secret store, if yes more details
ca_path: What is the recommended location? How do I specify intermediate CA path?

**Reply by xiajingchun** [reply]
Got it. There's really no much limitation for this. You should be able to use secret store to save all these files as long as they can be mapped to file pathes and can be accessed from within containers.
Not sure about intermediate CA, to my understanding, you don't need it if root CA certificate is available.
